src/include/mpir_misc.h

```diff
@@ -56,6 +56,11 @@ extern const char MPII_Version_F77[] MPICH_API_PUBLIC;
 extern const char MPII_Version_FC[] MPICH_API_PUBLIC;
 extern const char MPII_Version_custom[] MPICH_API_PUBLIC;
 
+int MPIR_Init(int *argc, char ***argv);
+int MPIR_Init_thread(int *argc, char ***argv, int user_required, int *provided);
+int MPIR_Finalize(void);
+int MPIR_Abort_impl(MPIR_Comm * comm_ptr, int errorcode);
```

Comment:
just curious, why `_impl` for abort, but none of the other functions?

Reply:
This is the current inconsistency. My plan is to consistently use `_impl`, will do it in a separate PR.